lib/api/src/sys/native.rs

```diff
@@ -87,15 +94,110 @@ macro_rules! impl_native_traits {
                     }
                     rets_list.as_mut()
                 };
-                unsafe {
-                    wasmer_vm::wasmer_call_trampoline(
-                        store.as_store_ref().signal_handler(),
-                        anyfunc.vmctx,
-                        anyfunc.call_trampoline,
-                        anyfunc.func_ptr,
-                        args_rets.as_mut_ptr() as *mut u8,
-                    )
-                }?;
+
```

Comment:
Can we add a new comment linking to the asyncify refactor afterwards:

```
// TODO: issue #XYZ 
```

Reply:
done